draft-ietf-tls-tls13.md

```diff
@@ -3951,6 +3949,22 @@ by IANA
   "Recommended": ecdsa_secp256r1_sha256, ecdsa_secp384r1_sha384,
   rsa_pss_sha256, rsa_pss_sha384, rsa_pss_sha512, ed25519.
 
+-  TLS ProtocolVersionID Registry: This registry formalizes the definition
+  of values previously labeled ProtocolVersion values in {{RFC5246}}. The
+  registry [shall be/ has been] initially populated with the values defined
+  as each ProtocolVersion value applicable to past SSL and TLS specifications.
+  All values with the first byte in the range 0-2 SHALL be marked as reserved
+  for SSL 2.0, in addition to { 2, 0 }, and are not available for use.
```

Comment:
Please reserve a portion of this space for experimentation, in the same way that we reserved space in cipher suites and extensions, etc...